wofi: 1.2.3 -> 1.2.4

Emantor opened the pull request:
###### Motivation for this change

Version bump.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Emantor committed:
wofi: 1.2.3 -> 1.2.4

Version bump.

r-rmcgibbo commented:
Result of `nixpkgs-review pr 112274` at f824cc12 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/3825149a37eb9dcd99d09145272ae5a7">wofi</a></li></li>
  </ul>
</details>
<details>
  <summary>1 warning produced:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/wofi/default.nix:28:5:
```
   |
28 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

r-rmcgibbo commented:
Result of `nixpkgs-review pr 112274` at f824cc12 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/21614dace4dac6bb29a7c515f6f33e06">wofi</a></li></li>
  </ul>
</details>
<details>
  <summary>1 warning produced:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/misc/wofi/default.nix:28:5:
```
   |
28 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

Emantor committed:
wofi: adjust license to gpl3Only

[1] does not have the "or any later version" suffix, so wofi is
gpl3Only.

[1]: https://hg.sr.ht/~scoopta/wofi/browse/COPYING.md?rev=tip
